examples/step-9/doc/intro.dox

```diff
@@ -66,24 +67,26 @@ for the derivatives in the perpendicular direction, and consequently
 if $u$ is discontinuous at a point on the inflow boundary, then this
 discontinuity will simply be transported along the streamline of the
 wind field that starts at this boundary point.
-
 These discontinuities lead to numerical instabilities that
 make a stable solution by a standard continuous finite element discretization
-impossible. We will thus use the streamline diffusion stabilized
-formulation, in which we test the equation with test functions $v +
-\delta \beta\cdot\nabla v$ instead of $v$, where $\delta$ is a
+impossible.
+
+A standard approach to address this difficulty is the "<i>streamline
+upwind Petrov Galerkin</i>" (SUPG) method, sometimes also called the
```

Comment:
```suggestion
A standard approach to address this difficulty is the <em>"streamline-upwind
Petrov-Galerkin"</em> (SUPG) method, sometimes also called the
```
Doxygen for whatever reason renders the HTML tags literally when they are in quotes.